### PR TITLE
Fix #66: Create new closure to set user permission

### LINKED ISSUE #66: Create user permissions properties closure
Create new properties file contains permissions for users

### PR DESCRIPTION
**Pull request by alexsedova**

### CODE REVIEW

File: dockerizeit/master/permissions.properties [hunk 1]
```diff
@@ -0,0 +1,12 @@
+user1 {
+		userId = hudson.security.ACL.ANONYMOUS_USERNAME
+		permissions = [
+			"hudson.model.Computer.Create",
```
Comment (Andrey9kin) [inline dockerizeit/master/permissions.properties]: Don't use strings - use actual classes, i.e. remove quotes then you don't need to do Permission.fromId(it) - you could simply say - it. But make sure to import 'import hudson.security.Permission' and 'import hudson.model.Computer' in the properties file

Comment (alexsedova) [inline dockerizeit/master/permissions.properties]: I'm not sure to understand what to do you mean import  'import hudson.security.Permission' and 'import hudson.model.Computer'  in properties file?

Comment (Andrey9kin) [inline dockerizeit/master/permissions.properties]: Skip that - you don't need to import them if you specify the full path to the class. Late night commenting is no good

What I mean is we should do it as it is done for the slaves - look for mode and retention

https://github.com/Praqma/JenkinsAsCodeReference/blob/master/dockerizeit/master/README.md#slaves

Also, do you consider 'Issue66' as a good commit message? Or what is the thinking behind that?
Where is documentation update

Comment (alexsedova) [inline dockerizeit/master/permissions.properties]: The commit message should be as usual "Fix #66 : ..." . May be I accidentally did copy/paste the branch name...Documentation will fix.

**Pull request merged**